GIT_WORKFLOW.md

```diff
@@ -1 +1,40 @@
-Write based on: https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow
\ No newline at end of file
+# Git Workflow
+
+The Economist Digital Solutions teams use git for version control following the [Forking  Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow).
+
+The advantage of the Forking Workflow is that contributions can be integrated without the need for everybody to push to a single central repository. Developers push to their own repositories, and only maintainer teams can push to the official repository. This allows  maintainers to accept commits from any developer without giving them write access to the official codebase. This approach scales easily and encourages sharing and collaboration on code bases across teams with minimal overhead required to manage merging and branching.
+
+## Forking Workflow Overview
+
+With the Forking Workflow every developer has their own private repository instead of one central repo. This means that each developer has two git repositories for a single project: a personal one and a public one.
+
+## Setting Up Your Fork
+
+When you're ready to collaborate on a project, rather than cloning the project, create a Fork of the project. Github provide a link to create a Fork on the main page of the project's Github Repo. This creates a person version of the repo associated with your account only, where you can control who can push to it.
+
+After creating your fork, perform a git clone of the Fork to get a copy onto your local machine. This serves as your private development environment, just like in the other workflows.
+
+If you want to regularlly pull the latest code from the main project into your Fork, add the project as an additional remote for your local project. 
+
+`git remote add upstream (upstream-github-url)`
+
+You can review the remotes associated with your project by running:
+
+`git remote -v`
+
+Before making changes to your code, ensure you have the latest changes from the main project by doing a git pull from that remote.
+
+`git pull upstream master`
+
+If you've made commits to your local code and want to then incorporate changes in the main repo, you can use git rebase. With rebase, you can take all the changes that were committed on the main repo and replay them on another one into your local git history. This helps ensures there are no merge conflicts when you submit a pull request.
+
+`git pull -r upstream master`
+
+
+## Submitting PRs
+
+The Forking Workflow typically follows a branching model. This means that complete feature branches will be purposed for merge into the original project's repository.
+
+For feature development work tied to specific Jira stories, it is recommended to create a branch on your Fork with the Jira ticket number in the branch. This automates integration between Jira and Github for tracking feature progress.
```

Comment:
Can we put this up where we create a branch on the fork before merging it with Jira ticket numbers ? Or are devs allowed to commit directly to their forks ? Here you mention branching ?

Reply:
Yes you can commit directly to your fork. You can do all your work in the master branch of your fork and put PRs from your fork's master branch to the main repo's master branch. For feature development, we create a branch on our own fork with the jira ticket number and then compare that branch to the main repo's master branch when it's ready for peer review. Did that clarify things?